src/main/java/ellie/command/DeleteCommand.java

```diff
@@ -26,9 +26,9 @@ public DeleteCommand(int index) {
      *
      * @param tasklist The TaskList to be operated on by the command.
      */
-    public void run(TaskList tasklist) {
-        tasklist.deleteTaskIndex(index);
-        return;
+    public String runAndReturnResponse(TaskList tasklist) {
+        String response = tasklist.deleteTaskIndex(index);
+        return response;
```

Comment:
Since this is an overridden method inherited from the parent class, you may consider using `@inheritDoc` tag, which allows you to reuse the header comment from the parent method but with further modifications.